protocol/communities/manager.go

```diff
@@ -2571,6 +2571,32 @@ func (m *Manager) GetCommunityTokens(communityID string, chainID int) ([]*Commun
 }
 
 func (m *Manager) AddCommunityToken(token *CommunityToken) error {
+
+	community, err := m.GetByIDString(token.CommunityID)
+	if err != nil {
+		return err
+	}
+	if community == nil {
+		return ErrOrgNotFound
+	}
+
+	tokenMetadata := &protobuf.CommunityTokenMetadata{
+		ContractAddresses: map[uint64]string{uint64(token.ChainID): token.Address},
+		Description:       token.Description,
+		Image:             token.Base64Image,
```

Comment:
For now at least we will not support minting ERC20 token, but probably in the future (?) /cc @John-44 
Maybe it makes sense to add a `type` here so we can check if it's an ERC721 or an ERC20
I know that we can check it onchain as well by asking the contract, but that would save found roundrip to the chain to query that data.

Same goes for `symbol` as well.

Reply:
I think it makes sense to add the type if we have a different view/workflow depending on the type. The symbol is probably important too, more than the description even

Reply:
@0x-r4bbit @jrainville we won't support minting ERC20 tokens in 0.11 but we may start work on adding support for this in either 0.12 or 0.13 e.g. there is a good likelihood we will end up adding support for minting ERC20 tokens before MVP.

Reply:
@0x-r4bbit @jrainville I added symbol and type.